demo/index.html

```diff
@@ -118,11 +118,12 @@ <h1>Vbrick SDK Embed Test</h1>
 					</div>
 				</form>
 				<div>
-					<b>Status: </b><span id="status"></span>
+					<div><b>Status: </b><span id="status"></span></div>
```

Comment:
Does this refer to Webcast Status? If so can we change its name?

Reply:
Its just a label.
This PR is not touching it, just needed to wrap the div.